crates/providers/http-server/wit/provider-httpserver.wit

```diff
@@ -1,5 +1,7 @@
 package wasmcloud:provider-http-server;
 
 world provider-http-server {
+    /// todo(vados-cosmonic): bus/host no longer exists, this provider must be rewritten
+    /// to only use wasi:http
```

Comment:
nitty nit nit: we did talk about it previously, due to the complexity of wasi:http this will be a _wasi:http-like_ contract